lib/src/facts/openbsd/virtualization_resolver.cc

```diff
@@ -22,4 +22,10 @@ namespace facter { namespace facts { namespace openbsd {
         return {};
     }
 
+    string virtualization_resolver::get_cloud_provider(collection& facts)
+    {
+        // TODO Implement this
+        return "";
```

Comment:
Is there something more explicit we can return here other than empty string so that it's clear this resolver hasn't been implemented yet? Or is this pattern pretty standard at the moment for facter?